requirements.txt

```diff
@@ -28,7 +35,11 @@ python-dotenv==0.15.0
 python-editor==1.0.4
 requests==2.25.1
 six==1.15.0
+slack-sdk==3.19.2
+slackclient==2.9.4
```

Comment:
For some reason this isn't getting installed when you submitted this to Learn which is why the error messages showed.